packages/test/test-version-utils/src/versionUtils.ts

```diff
@@ -438,13 +438,39 @@ function calculateRequestedRange(
 	}
 
 	// If the base version is a public version and `adjustPublicMajor` is false, then we need to ensure that we
-	// calculate N-1 as he previous major release, regardless if it is public or internal.
+	// calculate N-1 as the previous major release, regardless if it is public or internal.
 	// Currently, this case only applies to calculating N-X for 2.0.0.
 	// TODO: This is a temporary solution and we need to entirely rewrite this function to handle the changes the version schemas. See ADO:8198.
 	if (adjustPublicMajor === false && version.major > 1) {
-		// In this case, we can pretend that 2.0 is RC6 and calculate the range as if it were an internal version.
-		const internalSchemeRange = internalSchema("2.0.0", "6.0.0", "rc", requested);
-		return internalSchemeRange;
+		if (version.minor < 10) {
+			// If 2.0 <= N < 2.10, then we can pretend that N is RC6 and calculate the range as if it were an internal version.
```

Comment:
Just trying to add as much clarity as possible about potential confusion points in this function for people that have to understand it.
```suggestion
			// If 2.0 <= N < 2.10, then we can pretend that N is RC6 (*which doesn't exist*) and calculate the range as if it were an internal version.
```